packages/upload/src/ajax.ts

```diff
@@ -0,0 +1,91 @@
+import type {
+  ElUploadProgressEvent,
+  ElUploadRequestOptions,
+  ElUploadAjaxError,
+} from './upload'
+
+function getError(action: string, option: ElUploadRequestOptions, xhr: XMLHttpRequest) {
```

Comment:
```suggestion
function getError(action: string, option: ElUploadRequestOptions, xhr: XMLHttpRequest) : ElUploadAjaxError {
```
Is it ok do that ? Learning TypeScript as I review the PR.